lib/actions/logs.coffee

```diff
@@ -42,14 +30,18 @@ module.exports =
 	]
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.logs.subscribe params.uuid, {
-			history: options.num or LOGS_HISTORY_COUNT
-			tail: options.tail
-		}, (error, message) ->
-			return done(error) if error?
-			if _.isArray(message)
-				_.each message, (line) ->
+		if options.tail?
+			resin.logs.history(params.uuid).then (lines) ->
+				for line in lines
+					console.log(line.message)
+			resin.logs.subscribe(params.uuid).then (logs) ->
+				logs.on 'line', (line) ->
+					console.log(line.message)
+				logs.on 'error', (error) ->
```

Comment:
Remove `throw error` as well, since Capitano takes care of error handling if it's passed an Error instance to `done`.